SU-SOE/stanford_bean_types
# Featured Video Styling SOE-3561 and SOE-3695

# READY FOR REVIEW

# Summary
- SOE Featured paragraph type video styling

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for the release.

# Steps to Test
- checkout branch `7.x-2.x-dev-SOE-3561`
- `drush cc css-js`
- go to this page to see the style changes `catalog-patterns/blocks-and-beans/featured-content-blocks`
- make the fullwidth block on this page a video/image and add a video link if it's not already like that.
- see this page as an example for video content to copy https://eng-dev.stanford.edu/catalog-patterns/blocks-and-beans/featured-content-blocks
- make the test blocks on the homepage a video/image featured block as well and test out the changes there.
- if you switch back and forth between branches you will see the difference in the new styles.
- firefox and IE bugs are being handled here: 
https://stanfordits.atlassian.net/browse/SOE-3671 so no need to look at that in this PR.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3561
https://stanfordits.atlassian.net/browse/SOE-3499

## Related PRs

## More Information

## Folks to notify

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
